doc/onboarding.md

```diff
@@ -5,20 +5,17 @@ onboarding session.
 
 ## One week before the onboarding session
 
-* Confirm that the new Collaborator is using two-factor authentication on their
-  GitHub account. Unless two-factor authentication is enabled, do not give an
-  account elevated privileges such as the ability to land code in the main
-  repository or to start continuous integration (CI) jobs.
+* If the new Collaborator is not yet a member of the nodejs GitHub organization,
```

Comment:
A nit: nodejs -> Node.js?

Reply:
Or `[@nodejs](https://github.com/nodejs)`: [@nodejs](https://github.com/nodejs)
To correspond to @nodejs (which isn't available in *.md files).